src/toady/resolve_service.py

```
@@ -0,0 +1,223 @@
+            owner: Repository owner.
+            repo: Repository name.
+            pull_number: Pull request number.
+            thread_id: Thread ID to validate.
+
+        Returns:
+            True if the thread exists, False otherwise.
+        """
+        try:
+            # Query to check if thread exists in the PR
+            query = """
+            query CheckThreadExists($owner: String!, $repo: String!, $number: Int!, $threadId: ID!) {
```

Ruff (0.11.9)

Line too long (101 > 88)

(E501)